FESS-GUI/serial.cpp

```diff
@@ -0,0 +1,250 @@
+// Custom Libraries --------------------------------------------------
+#include "serial.h"
+
+//--------------------------------------------------------------------
+// Serial Constructors
+//--------------------------------------------------------------------
+
+Serial::Serial()
+{
+    device = new QSerialPort("/dev/pts/1");
```

Comment:
The QSerialPort and the values (baud rate, parity, etc.) are good candidates for some constructor-based dependency injection.